app/components/agenda/agenda-header/agenda-version-actions.js

```diff
@@ -50,13 +50,14 @@ export default class AgendaAgendaHeaderAgendaVersionActions extends Component {
   @tracked designAgenda = null;
   @tracked lastApprovedAgenda = null;
 
+  @tracked isClosed = false;
```

Comment:
Similar to previous comments, you can define a `get isFinalMeeting() { ... }` and use that instead of introducing the tracked property `isClosed`.